modules/dashboard/src/index.js

```diff
@@ -5,13 +5,18 @@ import * as serviceWorker from './serviceWorker';
 
 console.log(`Starting dashboard in env: ${JSON.stringify(process.env,null,2)}`)
 
-const apiUrl = process.env.REACT_APP_API_URL || `http://localhost:9999`
-const hubUrl = process.env.REACT_APP_HUB_URL || `http://localhost:8080`
-const ethUrl = process.env.REACT_APP_ETH_URL || `http://localhost:8545`
+const urlPrefix = process.env.REACT_APP_PUBLIC_URL || `/dashboard`
+const publicUrl = `${window.location.origin}${urlPrefix}`
+const apiUrl = process.env.REACT_APP_API_URL || `${publicUrl}/api/dashboard`
+const hubUrl = process.env.REACT_APP_HUB_URL || `${publicUrl}/api/hub`
+const ethUrl = process.env.REACT_APP_ETH_URL || `${publicUrl}/api/eth`
 
-ReactDOM.render(<Index ethUrl={ethUrl} hubUrl={hubUrl} apiUrl={apiUrl}/>, document.getElementById('root'));
+ReactDOM.render(
+  <Index publicUrl={urlPrefix} ethUrl={ethUrl} hubUrl={hubUrl} apiUrl={apiUrl}/>,
+  document.getElementById('root')
+);
 
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: http://bit.ly/CRA-PWA
-serviceWorker.unregister();
+serviceWorker.register();
```

Comment:
I updated the serviceWorker to use `window.location.origin` instead of `process.env.PUBLIC_URL` so it'll work on both rinkeby & mainnet. Tentatively turned it on during testing to make sure that my changes didn't break anything but I don't think this is necessary. We can keep the serviceWorked deactivated if it ends up causing problems.